### PR TITLE
Document update for ROCm3.9

### PR DESCRIPTION
**Pull request by sunway513**

### CODE REVIEW

**Comment by rsanthanam-amd** [comment]
jenkins : retest rocm-cpp please

**Review by mvermeulen — APPROVED** [review]
The changes made look fine.

I do note however that the "The ROCm3.1 release changes the installation directory structure." sentence seems old as the overall ROCm instructions still say we don't support updates, e.g. "A fresh and clean installation of AMD ROCm v3.9 is recommended. An upgrade from previous releases to AMD ROCm v3.9 is not supported."  So we might want to change that note to something like:

*NOTE*: ROCm install instructions recommend a purge and reinstall of ROCm rather than upgrading from previous release.

**Comment by sunway513** [comment]
Thanks @mvermeulen , that's a good point. Let me rephrase the wording there.
